backend/src/database/seeds/08_picker_options.js

```diff
@@ -115,6 +116,14 @@ exports.seed = function (knex) {
       definition: { dropDownValues: projectType },
       associated_form: tables.projects,
     },
+
+    {
```

Comment:
shouldn't you indicate in the changelog that you need to update the seeds for picker options